src/NexusMods.App.UI/Resources/Language.resx

```diff
@@ -681,4 +681,31 @@
     <data name="LoadoutView_Title_Installed_Mods" xml:space="preserve">
         <value>Installed Mods</value>
     </data>
+    <data name="FileOriginPage_SwitchView" xml:space="preserve">
+        <value>Switch View</value>
+    </data>
+    <data name="FileOriginPage_SwitchView_ToolTip" xml:space="preserve">
+        <value>Show or hide mod page groups</value>
+    </data>
+    <data name="FileOriginsPage_FromDrive_ToolTip" xml:space="preserve">
+        <value>Import a mod from your computer</value>
+    </data>
+    <data name="FileOriginsPage_FromNexus_ToolTip" xml:space="preserve">
+        <value>Open Nexus Mods</value>
+    </data>
+    <data name="TopBar_Forward_ToolTip" xml:space="preserve">
+        <value>Forward</value>
+    </data>
+    <data name="TopBar_Back_ToolTip" xml:space="preserve">
+        <value>Back</value>
+    </data>
+    <data name="TopBarActions__LOG_IN_ToolTip" xml:space="preserve">
+        <value>Connect your Nexus Mods account</value>
+    </data>
+    <data name="LoadoutGridView__Remove_ToolTip" xml:space="preserve">
+        <value>Remove the selected mod(s) from this loadout</value>
+    </data>
+    <data name="LoadoutGridView__View_Files_ToolTip" xml:space="preserve">
```

Comment:
data name "LoadoutGridView__View_Files_ToolTip" has an empty value?

Reply:
Not decided on a tooltip yet but added this to the resource and code for completeness